node/rpc/handlers/status.go

```diff
@@ -14,8 +18,13 @@ func (s StatusHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	//nolint:errcheck
 	jsonStatus, err := json.Marshal(s)
 	if err != nil {
-		jsonStatus, _ = json.Marshal("Error: Could not marshal the status. " + err.Error())
+		errorMessage := "Could not marshal the status. " + err.Error()
+		jsonStatus, _ = json.Marshal(errorMessage)
```

Comment:
I have a question that goes beyond the scope of this PR: is this and the other end-pooints meant to adhere to the JSON RPC conventions? https://www.jsonrpc.org/specification#error_object

Reply:
If you do a w.Write() with an error message as the input without writing the header, it'll write a default 200 status header which can sometimes be inaccurate depending on the nature of the error. In this case, I would say that this is an InternalServerError that shouldn't be exposed to the user.

Reply:
So what i'm saying is - in this case - if error marshaling status, don't write a message, just write a header with the status code.